Makefile

```diff
@@ -404,3 +404,11 @@ integration-tests: all testrunner
 #     integration-tests
 #
 	RUNNER_ARGS="-parallel" ./test/run-integration-tests.sh
+
+netlify-site-preview:
+	@mkdir -p site-build
```

Comment:
We can delete it, but this is only intended to run in Netlify CI, and it won't work anywhere else without a bunch of dependencies. If you need to build locally, the best way to do it is via website repo.